src/components/admin/users/user-management.tsx

```
@@ -0,0 +1,517 @@
+    AlertDialog,
+    AlertDialogAction,
+    AlertDialogCancel,
+    AlertDialogContent,
+    AlertDialogDescription,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogTitle,
+} from "@/components/ui/alert-dialog";
+import { KeycloakSyncDialog } from "./keycloak-sync-dialog";
+
+interface User {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'User'. Consider to delete it or rename it.

'User' is defined here:


(lint/suspicious/noRedeclare)